mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DaoBase.java

```diff
@@ -285,4 +289,19 @@ protected static void throwIfProtocolVersionV3(MapperContext context) {
               NullSavingStrategy.class.getSimpleName(), NullSavingStrategy.DO_NOT_SET.name()));
     }
   }
+
+  protected static <EntityT> void throwIfKeyspaceMissing(
+      EntityHelper<EntityT> helper, Session session) {
+    if (helper.getKeyspaceId() == null && !session.getKeyspace().isPresent()) {
+      throw new MapperException(
+          String.format(
+              "Missing keyspace. Suggestions: use SessionBuilder.withKeyspace() "
+                  + "when creating your session, specify a default keyspace on %s with @%s"
+                  + "(defaultKeyspace), or use a @%s method with a @%s parameter",
+              helper.getEntityClass(),
```

Comment:
In similar messages (see `replaceKeyspaceAndTablePlaceholders`), I've used the entity's simple name. My reasoning was that it would be pretty obvious from the context which class it is.
I don't mind either way but we should be consistent, if you think it's worth having the package we'll modify the other message.

Reply:
Fixed that to use `getSimpleName()` :+1: